code/BasicFieldsTestPage.php

```diff
@@ -71,7 +71,7 @@ class BasicFieldsTestPage extends TestPage
         'TimeHTML5' => 'Time',
         'ToggleCompositeTextField1' => 'Varchar',
         'ToggleCompositeDropdownField' => 'Varchar',
-        'Validated' => 'Text',
+        'Validated' => 'Int',
```

Comment:
What a terrible name for this field lol. I was like "shouldn't that be a bool?" and had to look at how it's being used.

Don't bother changing that though, that's unrelated.